### PR TITLE
Update plugin.json to make template more "marketplace-ready"

### PR DESCRIPTION
**Pull request by haardikdharma10**
#### Summary

Adds default icon to `/assets` folder and updates `plugin.json`

#### Ticket Link

Partially Fixes: #132

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -2,6 +2,10 @@
     "id": "com.mattermost.plugin-starter-template",
     "name": "Plugin Starter Template",
     "description": "This plugin serves as a starting point for writing a Mattermost plugin.",
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-starter-template",
+    "support_url": "https://github.com/mattermost/mattermost-plugin-starter-template/issues",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-starter-template/releases/tag/vX.X.X",
```
Comment (hanzei) [inline plugin.json:7]: Should we preemptively fill that out?
```suggestion
    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-starter-template/releases/tag/v0.1.0",
```

**Review by hanzei — APPROVED** [review]
LGTM :+1: 

One non blocking question

**Comment by haardikdharma10** [comment]
Thanks @hanzei!

**Review by jfrerich — APPROVED** [review]
LGTM

**Comment by hanzei** [comment]
Tested locally. Icon is shown and link works.

**Pull request merged**